asv_bench/benchmarks/timeseries.py

```diff
@@ -394,33 +394,42 @@ def time_dup_string_tzoffset_dates(self, cache):
 
 class DatetimeAccessor:
 
-    params = [None, "US/Eastern", "UTC", dateutil.tz.tzutc()]
-    param_names = "tz"
-
-    def setup(self, tz):
+    params = (
+        [None, "US/Eastern", "UTC", dateutil.tz.tzutc()],
+        ["%Y-%m-%d %H:%M:%S.%f%z", "%Y-%m-%d %H:%M:%S%z"],
+        ["T", "S", "NS"],
+    )
+    param_names = ["tz", "fmt", "frequency"]
+
+    def setup(self, tz, fmt, frequency):
         N = 100000
-        self.series = Series(date_range(start="1/1/2000", periods=N, freq="T", tz=tz))
+        self.series = Series(
+            date_range(start="1/1/2000", periods=N, freq=frequency, tz=tz)
+        )
 
-    def time_dt_accessor(self, tz):
+    def time_dt_accessor(self, *args):
```

Comment:
Can you use explicit parameters here instead of args?

Reply:
Yeah, I'll fix that.